spec/models/spree/order_spec.rb

```diff
@@ -54,7 +54,7 @@
     end
 
     it "does nothing when the line item is not found" do
-      p = create(:simple_product)
+      p = build_stubbed(:simple_product)
       subject.set_variant_attributes(p.master, { 'max_quantity' => '3' }.with_indifferent_access)
```

Comment:
It's weird that this test does not have any assertions.

Reply:
Maybe it's a no-blow-up test?
If the call raises it will break the test?
I am ok with improving the spec or deleting the specs :+1:

Reply:
Sure, let's do it in a separate PR.